src/Button.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./button.css";
+
+// HINT: Use this in ButtonProps to determine how the button is styled
+export enum ButtonType {
+  green="button-green",
+  white="button-white",
+  clear="button-clear",
+  clearAlt="button-clearAlt",
+}
+
+// HINT: Use this in ButtonProps to determine how the button text is styled
+export enum ButtonTextType {
+  white="button-text-white",
+  green="button-text-green",
+  grey="button-text-grey",
+}
+
+interface ButtonProps {
+    text: string;
+    buttonType: ButtonType;
```

Comment:
Good work working with enums. Here's an easier way: rather than making those enumerations map keys to strings that correspond to the css selector, use `keyof typeof ButtonType` instead where ButtonType might look something like 
`export enum ButtonType {
  "button-green",
  "button-white",
  "button-clear",
  "button-clearAlt"
}`